src/SFA.DAS.EAS.Account.Api.UnitTests/SFA.DAS.EAS.Account.Api.UnitTests.csproj

```diff
@@ -100,6 +101,10 @@
       <Project>{05D7C094-E2C9-4CCD-B0D7-85D263254B26}</Project>
       <Name>SFA.DAS.EAS.TestCommon</Name>
     </ProjectReference>
+    <ProjectReference Include="..\SFA.DAS.EmployerFinance\SFA.DAS.EmployerFinance.csproj">
+      <Project>{C5880F73-59D6-4C59-A1A2-DBA38D13E8DC}</Project>
+      <Name>SFA.DAS.EmployerFinance</Name>
+    </ProjectReference>

```

Comment:
EAS Api shouldn't have a direct reference to FInance.csproj else we won't be able to split to separate codebases.

Reply:
remove the ref